mount/mount.go

```diff
@@ -36,7 +36,14 @@ const (
 // Interface defines the set of methods to allow for mount operations on a system.
 type Interface interface {
 	// Mount mounts source to target as fstype with given options.
+	// options MUST not contain sensitive material (like passwords).
 	Mount(source string, target string, fstype string, options []string) error
+	// MountSensitive is the same as Mount() but this method allows
+	// sensitiveOptions to be passed in a separate parameter from the normal
+	// mount options and ensures the sensitiveOptions are never logged. This
+	// method should be used by callers that pass sensitive material (like
+	// passwords) as mount options.
+	MountSensitive(source string, target string, fstype string, options []string, sensitiveOptions []string) error
```

Comment:
do we expect implementations of `mount.Interface` other than the fake/linux/windows/unsupported mounters provided by this package? if so, adding this method to an existing interface makes this version of k8s.io/utils incompatible with those implementations

Reply:
looks like the mount_windows.go implementation needs updating as well

Reply:
> do we expect implementations of mount.Interface other than the fake/linux/windows/unsupported mounters provided by this package? if so, adding this method to an existing interface makes this version of k8s.io/utils incompatible with those implementations

My understand is, no. Based on the Kubernetes consumers, the callers all first construct one of the `Mounters` provided by this package (fake/linux/windows/unsupported). There is little value in using the other methods provided by the library with a custom mounter.

Updated `mount_windows.go` implementation.